src/context/WalletProvider/KeepKey/setupKeepKeySdk.ts

```diff
@@ -0,0 +1,20 @@
+import { KeepKeySdk } from '@keepkey/keepkey-sdk'
+
+export const setupKeepKeySDK = async () => {
+  let serviceKey = window.localStorage.getItem('@app/serviceKey')
```

Comment:
consts over let through for consistency, please

Reply:
updated